docs/source/guides/custom_types_and_type_inference.ipynb

```diff
@@ -49,8 +49,8 @@
     "\n",
     "class UPCCode(LogicalType):\n",
     "    \"\"\"Represents Logical Types that contain 12-digit UPC Codes.\"\"\"\n",
-    "    pandas_dtype = 'category'\n",
-    "    backup_dtype = 'str'\n",
+    "    primary_dtype = 'category'\n",
```

Comment:
Not sure when the `pandas_dtype` to `primary_dtype` was decided, but I like it. Definite a better description!!

Reply:
We decided to make that change when we were reworking the way dtypes were handled. "Primary" fits better since this is used for both pandas and Dask. Glad you approve!